packages/theme/stories/introduction.stories.mdx

```diff
@@ -62,14 +50,20 @@ A simple example will help to illustrate some of the key concepts:
+### Theme CSS + Component CSS
+
+The styling of our UI Toolkit componentry is defined in the following two places, which operate together at runtime:
+
+- Theme styles - a Toolkit theme consists of CSS Variables plus a small number of reusable _utility_ classes
+- Component styles - each component contributes its own stylesheet. Many of the CSS declarations in here will reference CSS variables defined in the theme i.e. the variables will be defined at the theme level and used at the component level.
 
-1. Theme styles - a Toolkit theme consists of CSS Variables plus a small number of reusable _utility_ classes
-2. Component styles - each component contributes its own stylesheet. Many of the CSS declarations in here will reference CSS variables defined in the theme i.e. the variables will be defined at the theme level and used at the component level.
+#### Theme styles
 
-### Theme styles
+A UI Toolkit theme is a collection of design tokens. These tokens are defined in Figma and have exact equivalents in code. In Figma, they are used to apply styling to all Figma components. In code, they are deployed as CSS Variables. The tokens fall into the following three categories:
 
-A UI Toolkit theme is a collection of design tokens. These tokens are defined in Figma and have exact equivalents in code. In Figma, they are used to apply styling to all Figma components. In code, they are deployed as CSS Variables. The tokens fall into three categories, foundations and characteristics, and an in-between palette. Foundation tokens are the primitive values for attributes like color, size, shadow etc. Characteristic tokens represent a slightly higher level abstraction - they are groupings of related attributes typically used together, that represent common aspects of a UI component or application. Characteristics fall between the simple foundational tokens and component-specific tokens defined at the component level (described below). They represent general groupings of related style attributes that typically apply across multiple components. Lastly, the palette is a carefully selected group of color tokens which are aliased to give semantic meaning; allowing for a more consistent look and feel when used by characteristics, rather than foundation colors directly. For example, actionable, selectable, and editable tokens all use 'interact' palette tokens, which UITK has defined to be blue, instead of using the color token itself. If the theme updated and interactable components needed to be red; this would only have to be updated in one place, the palette, instead of within each characteristic.
+- foundations - Foundation tokens are the primitive values for attributes like color, size, shadow etc. Characteristic tokens represent a slightly higher level abstraction - they are groupings of related attributes typically used together, that represent common aspects of a UI component or application.
```

Comment:
This is more for my knowledge. Here the bullet points don't begin with a capital letter, but there is one after the hyphen. Is this a pattern we will use now?

Reply:
No, this was an oversight on my part after reformatting the paragraph as a bulleted list.